source/main/physics/BeamFactory.cpp

```diff
@@ -1077,21 +1077,21 @@ void ActorManager::UpdateActors(Actor* player_actor, float dt)
             actor->ar_vehicle_ai->update(dt, 0);
 #endif // USE_ANGELSCRIPT
 
-        if (actor->ar_sim_state == Actor::SimState::LOCAL_SIMULATED)
+        if (actor->ar_sim_state < Actor::SimState::LOCAL_SIMULATED)

```

Comment:
This must be wrong. LOCAL_SIMULATED is (0), so even `<=` wouldn't make sense.

Also I generally dislike `<=>=` with enums, just duplicate the call in different branches please.

Reply:
> I generally dislike <=>= with enums

Me too, that's why I fucked this up in the first place. :blush: